### PR TITLE
activate resolveLoops

### PR DESCRIPTION
**Pull request by vwaurich**
this PR is just for testing on hudson. no pushing.

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2016-12-16_12-04-29](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3582/).

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2016-12-19_11-42-30](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3600/).

File: Compiler/BackEnd/BackendDAEUtil.mo [hunk 1]
```diff
@@ -7470,6 +7469,7 @@ algorithm
     // handle special flags, which disable modules
     disabledModules := deprecatedDebugFlag(Flags.NO_PARTITIONING, disabledModules, "clockPartitioning", "preOptModules-");
     disabledModules := deprecatedDebugFlag(Flags.DISABLE_COMSUBEXP, disabledModules, "comSubExp", "preOptModules-");
+    disabledModules := deprecatedDebugFlag(Flags.RESOLVE_LOOPS, enabledModules, "resolveLoops", "preOptModules+");
```
Comment (lochel) [inline Compiler/BackEnd/BackendDAEUtil.mo]: Shouldn't this disable `resolveLoops`?
`disabledModules := deprecatedDebugFlag(Flags.RESOLVE_LOOPS, disabledModules, "resolveLoops", "preOptModules-");`

File: Compiler/Util/Flags.mo [hunk 2]
```diff
@@ -364,7 +364,7 @@ constant DebugFlag INFO_XML_OPERATIONS = DEBUG_FLAG(96, "infoXmlOperations", fal
   Util.gettext("Enables output of the operations in the _info.xml file when translating models."));
 constant DebugFlag HPCOM_DUMP = DEBUG_FLAG(97, "hpcomDump", false,
   Util.gettext("Dumps additional information on the parallel execution with hpcom."));
-constant DebugFlag RESOLVE_LOOPS = DEBUG_FLAG(98, "resolveLoops", true,
+constant DebugFlag RESOLVE_LOOPS = DEBUG_FLAG(98, "resolveLoops", false,
```
Comment (lochel) [inline Compiler/Util/Flags.mo]: Is this flag necessary or can it just get removed?

Comment (lochel) [inline Compiler/Util/Flags.mo]: For testing purpose, you can also run Hudson on your branches without creating pull requests.

It is possible to deactivate all backend modules without special flags, e.g. `preOptModules-=resolveLoops`.

Comment (vwaurich) [inline Compiler/Util/Flags.mo]: Alright, I will consider this.

**Pull request closed without merging**